docs/reference/manifest/extendedpermission.md

```diff
@@ -0,0 +1,55 @@
+---
+title: ExtendedPermission element in the manifest file
+description: ''
+ms.date: 03/04/2020
+localization_priority: Normal
+---
+
+# `ExtendedPermission` element
+
+Defines an extended permission the add-in needs to work. The `ExtendedPermission` element is a child element of [ExtendedPermissions](extendedpermissions.md).
```

Comment:
We should reword this - Defines extended permission so the add-in has access to the API/a feature? Currently no features are usig this so we can limit this to just an API and update later or add both so we don't add later when a feature is added under it. But from what it is right now it seems likt the addin won't work if this isn't mentioned however it's only the API that wouldn't work. If the addin doesn't use the API then this element is not required,

Reply:
Reworded.